pinot-tools/src/main/java/com/linkedin/pinot/tools/segment/converter/PinotSegmentToAvroConverter.java

```diff
@@ -68,61 +67,4 @@ public void convert() throws Exception {
       }
     }
   }
-
-  /**
-   * Helper method to build Avro schema from Pinot schema.
-   *
-   * @param pinotSchema Pinot schema.
-   * @return Avro schema.
-   */
-  private Schema buildAvroSchemaFromPinotSchema(com.linkedin.pinot.common.data.Schema pinotSchema) {
```

Comment:
refactored this method to `AvroUtils.getAvroSchemaFromPinotSchema` to share across other modules.